bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/multihrefMetadata.js

```diff
@@ -424,6 +424,7 @@ pimcore.object.classes.data.multihrefMetadata = Class.create(pimcore.object.clas
             var types = {
                 number: t("objectsMetadata_type_number"),
                 text: t("objectsMetadata_type_text"),
+                date: t("objectsMetadata_type_date"),
```

Comment:
The translation key `objectsMetadata_type_date` doesn't exist. 
Please use the `date` translation key instead, which does exist already. 
The other translations keys which are prefixed with `objectsMetadata_type_` are kind of legacy and are just aliased here: https://github.com/pimcore/pimcore/blob/c62924ca5c3d0c482a71097cb0231aff143208d6/bundles/CoreBundle/Resources/translations/aliases.json#L25 so there's nothing wrong by just usinf `date` directly´.